.github/workflows/publish_docker_images_on_manual.yml

```diff
@@ -33,7 +32,8 @@ jobs:
 
       - name: Build and publish docker images
         run: |
-            python -m  tools.packaging_automation.publish_docker  \
-            --pipeline_trigger_type "${GITHUB_EVENT_NAME}" \
-            --tag_name ${{ github.event.inputs.tag_name }} \
-            --manual_trigger_type ${{ github.event.inputs.trigger_type }}
+          python -m  tools.packaging_automation.publish_docker  \
+          --pipeline_trigger_type "${GITHUB_EVENT_NAME}" \
+          --tag_name ${{ github.event.inputs.tag_name }} \
+          --manual_trigger_type ${{ github.event.inputs.trigger_type }} \
+          --github_ref ${GITHUB_REF}
```

Comment:
If you can compare two of the lines in this file
`--pipeline_trigger_type "${GITHUB_EVENT_NAME}"`
and 
`--github_ref ${GITHUB_REF}`
you will see that you are not being consistent with your use of quotation marks.

Reply:
You're right. Fixed I'll be more careful from now on